src/sentry/api/event_search.py

```diff
@@ -1390,9 +1399,9 @@ def key_transaction_expression(user_id, organization_id, project_ids):
     ]
 
 
-# When adding aliases to this list please also update
-# static/app/utils/discover/fields.tsx so that
-# the UI builder stays in sync.
+# When updating this list, also check if the following need to be updated:
+# - convert_search_filter_to_snuba_query
+# - static/app/utils/discover/fields.tsx FIELDS
```

Comment:
Can we keep the bit in this comment about "so that the UI builder stays in sync", we've had questions before about how the two stay in sync.

Reply:
Addressed in https://github.com/getsentry/sentry/pull/24530/commits/dad1e13ff94215913a55a64f116abd928da19542, thank you for the feedback.